### ISSUE TITLE
SendMail via Parse.Cloud.sendMail()

### ISSUE THREAD

**Issue by dblythy** [parse-community/parse-server#7089]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Current Limitation
I currently use a Mailgun emailAdapter on my Parse Servers. Yet in my cloud code whenever I want to send email, I have to call:

```
  const api_key = "";
  const domain = "";
  const mailgun = require("mailgun-js")({
    apiKey: api_key,
    domain: domain
    });
  const data = {
    from: "",
    to: obj.get("email"),
    subject: "",
    text: ",
    html: summary
    };
  mailgun.messages().send(data, function(error, body) {});
```

I could be doing this wrong, idk.

### Feature / Enhancement Description

It would be nice to have:

`Parse.Cloud.sendEmail(data)` and it inherit the sendEmail function from the emailAdapter. It would also support newer developers as they don't have to read their mail providers docs to learn how to send email, as the adapter handles it for them. If no email adapter is specified, throw an error.

**Comment by mman** [comment]
You can already do this easily like this:

```javascript
const { AppCache } = require('parse-server/lib/cache');

Parse.Cloud.afterLogin(async request => {
  const MailgunAdapter = AppCache.get('INSERT_YOUR_APPID_HERE').userController.adapter;

  const { object: user }  = request;
  const email = user.get('email');

    return MailgunAdapter.send({
      templateName: 'newLoginEmail',
      recipient: email,
      variables: { username: email, date: Date().toString() }
    });
  }
});
```

**Comment by mtrezza** [comment]
Thanks for suggesting.

It may be just a simple PR, but it could be a nice convenience method to simply call `Parse.Cloud.sendMail({...})` instead of having to reference the mail adapter manually, saving one or two lines of code.

**Comment by dblythy** [comment]
PR already on the way if you’re keen to review my code again @mtrezza ;)

**Comment by mtrezza** [comment]
Closed by #7096.
